pkg/models/terminal/terminal.go

```diff
@@ -200,8 +219,12 @@ func (t *terminaler) HandleSession(shell, namespace, podName, containerName stri
 	var err error
 	validShells := []string{"sh", "bash"}
 
+	stopChan := make(chan struct{})
+
 	session := &TerminalSession{conn: conn, sizeChan: make(chan remotecommand.TerminalSize)}
 
+	go wait.Until(session.sendHeartBeatPackage, time.Duration(3)*time.Second, stopChan)
```

Comment:
The default interval should be constant, I think the interval should be increased appropriately.

Reply:
Should we increase the interval such as 1s, 2s, 4s, 8s?